src/renderer/views/Watch/Watch.js

```diff
@@ -133,7 +133,8 @@ export default defineComponent({
       customErrorIcon: null,
       videoGenreIsMusic: false,
       /** @type {Date|null} */
-      streamingDataExpiryDate: null
+      streamingDataExpiryDate: null,
+      currentPlaybackRate: parseFloat(this.$store.getters.getDefaultPlayback)
```

Comment:
Can you please access the store in the created lifecycle hook, instead of in data. In the options API the only thing you are supposed to access at that point is the props, everything else is usually only ready when the created lifecycle hook fires.

I know we are slowly moving away from the options API but while we still have options API components we should still write their code correctly.